src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -265,6 +265,9 @@ enum AppStrings {
 		static let resetDescription = NSLocalizedString("Settings_Reset_Description", comment: "")
 
 		static let navigationBarTitle = NSLocalizedString("Settings_NavTitle", comment: "")
+
+		static let onImageDescription = NSLocalizedString("Settings_onImageDescription", comment: "")
+		static let offImageDescription = NSLocalizedString("Settings_offImageDescription", comment: "")
 	}
 
 	enum NotificationSettings {
```

Comment:
Please move the two string that you've added here.

Reply:
Renamed "Settings..." to "NotificationSettings..." to align with the Localizable.strings files

Reply:
This is not critical, but it would be nice if you'd move these two strings into ```NotificationSettings``` enum below, because that's where they belong. Sorry, my original comment was probably unclear.

Reply:
Ah apologies, understood, will fix that now.